KsApi/models/Backing.swift

```diff
@@ -91,8 +114,27 @@ extension Backing: EncodableType {
   }
 }
 
+extension Backing.PaymentSource: Argo.Decodable {
+  public static func decode(_ json: JSON) -> Decoded<Backing.PaymentSource?> {
+    return curry(Backing.PaymentSource.init)
+      <^> json <|? "expiration_date"
+      <*> (json <|? "id" <|> (json <| "id" >>- intToString))
```

Comment:
CJ merged the change to return a `String` instead of an `Int` so you should be able to remove this now! 🎉